modules/core/include/opencv2/core/mat.hpp

```diff
@@ -61,7 +61,7 @@ namespace cv
 //! @addtogroup core_basic
 //! @{
 
-enum { ACCESS_READ=1<<24, ACCESS_WRITE=1<<25,
+enum CV_EXPORTS_W_SIMPLE { ACCESS_READ=1<<24, ACCESS_WRITE=1<<25,
```

Comment:
CV_EXPORTS_W_SIMPLE doesn't work for enums.

All these flags are already in Python: `cv.ACCESS_READ`